docs/cli/get-started/wallet-connect.md

```diff
@@ -35,9 +35,12 @@ Notes:
 ### How it works (overview)
 
 - The CLI initializes a WalletConnect session using your `WALLET_CONNECT_PROJECT_ID`.
-- It prints a WC URI and displays a QR code in the terminal.
-- You open your wallet, scan the QR (or open the URI), and approve the session.
+- It prints a WalletConnect URI and displays a QR code in the terminal.
+- You approve the session:
+  - scan the QR code with your mobile phone, and a wallet app that supports WalletConnect should be opened automatically.
+  - or copy and paste the WalletConnect URI into a desktop wallet application or multisig wallet that supports URI input.
```

Comment:
"desktop wallet application or multisig wallet" - it's essentially the same thing